models/Flow.js

```diff
@@ -0,0 +1,18 @@
+import mongoose from "mongoose";
+
+const { Schema } = mongoose;
+
+const flowSchema = new Schema({
+  name: { type: String, required: true },
+  description: { type: String },
+  duration: {
+    hours: { type: Number },
+    minutes: { type: Number },
+  },
+  asanas: { type: Array },
+});
+
+const Flow =
+  mongoose.models.Flow || mongoose.model("Flow", flowSchema, "flows");
```

Comment:
Same applies here. Consider renaming the model name to lowercase letters in case I'm not wrong about my assumption of mongoose converting everything to lowercase.